apstra/stream_target.go

```diff
@@ -21,7 +21,7 @@ const (
 	EnvApstraStreamBasePort = "APSTRA_STREAM_BASE_PORT"
 
 	sizeOfAosMessageLenHdr = 2 // Apstra 'protoBufOverTcp' streaming includes a 16-bit length w/each protobuf
-	network                = "tcp4"
+	stNetwork              = "tcp4"
```

Comment:
Constant renamed because the `stream_target` client code was sitting on a global constant called `network`. Not very convenient.